aerolyzer/image_restriction_functions.py

```diff
@@ -50,14 +50,17 @@ def is_edited(self, created, modified):
     '''
     Purpose:        The purpose of this function is to determine whether or not the image
                     contains a direct landscape with sky and view.
-    Inputs:         dict exifData, ?
+    Inputs:         list of lists of lists of rgb values [red, green, blue]
     Outputs:        None
     Returns:        Boolean
     Assumptions:    N/A
     '''
-    def is_landscape(self, exifData):
-        pass
-        #UNKNOWN METHOD
+    def is_landscape(self, rgb):
+        [red, green, blue] = rgb
```

Comment:
this line is unnecessary, and creates an unnecessary new variable (mem usage). Access the values using the tuple notation. e.g. red_sky = rgb(0)[:len(rgb(0))/2] assuming rgb is a tuple of lists